assets/src/setup/pages/welcome/index.js

```diff
@@ -0,0 +1,151 @@
+								<path d="M34.2437 6.36438H26.9712C26.4693 6.36438 26.0618 5.9575 26.0618 5.45562C26.0618 4.95375 26.4687 4.54688 26.9712 4.54688H34.2437C34.7456 4.54688 35.1531 4.95375 35.1531 5.45562C35.1531 5.9575 34.7462 6.36438 34.2437 6.36438Z" fill="#005AF0" />
+								<path d="M34.2438 10.0011H26.9713C26.4694 10.0011 26.0619 9.59422 26.0619 9.09234C26.0619 8.59047 26.4688 8.18359 26.9713 8.18359H34.2438C34.7456 8.18359 35.1531 8.59047 35.1531 9.09234C35.1531 9.59422 34.7463 10.0011 34.2438 10.0011Z" fill="#005AF0" />
+								<path d="M34.2438 13.6378H26.9713C26.4694 13.6378 26.0619 13.2309 26.0619 12.7291C26.0619 12.2272 26.4688 11.8203 26.9713 11.8203H34.2438C34.7456 11.8203 35.1531 12.2272 35.1531 12.7291C35.1531 13.2309 34.7463 13.6378 34.2438 13.6378Z" fill="#005AF0" />
+								<path d="M36.9543 18.1819H24.2618C22.7468 18.1806 21.5187 16.9525 21.5175 15.4375V2.74437C21.5187 1.22937 22.7468 0.00125 24.2618 0H36.955C38.47 0.00125 39.6981 1.22937 39.6993 2.74437V15.4375C39.6981 16.9525 38.47 18.1806 36.955 18.1819H36.9543ZM24.2612 1.81813C23.75 1.81875 23.3356 2.23312 23.335 2.74437V15.4369C23.3356 15.9481 23.75 16.3625 24.2612 16.3631H36.9537C37.465 16.3625 37.8793 15.9481 37.88 15.4369V2.74437C37.8793 2.23312 37.465 1.81875 36.9537 1.81813H24.2612Z" fill="#005AF0" />
+								<path d="M36.9543 40H3.35247C1.83747 39.9988 0.609343 38.7706 0.608093 37.2556V22.7444C0.609343 21.2294 1.83747 20.0013 3.35247 20H36.9543C38.4693 20.0013 39.6975 21.2294 39.6987 22.7444V37.2556C39.6975 38.7706 38.4693 39.9988 36.9543 40ZM3.35247 21.8181C2.84122 21.8188 2.42684 22.2331 2.42622 22.7444V37.2556C2.42684 37.7669 2.84122 38.1812 3.35247 38.1819H36.9543C37.4656 38.1812 37.88 37.7669 37.8806 37.2556V22.7444C37.88 22.2331 37.4656 21.8188 36.9543 21.8181H3.35247Z" fill="#005AF0" />
+								<path d="M17.8806 34.5456C17.3787 34.5456 16.9718 34.1388 16.9718 33.6362V26.4244C16.9718 25.9225 17.3787 25.5156 17.8806 25.5156C18.0668 25.5156 18.2406 25.5719 18.3849 25.6681L18.3818 25.6663L23.8362 29.2719C24.0831 29.4369 24.2437 29.715 24.2437 30.03C24.2437 30.345 24.0831 30.6231 23.8393 30.7862L23.8362 30.7881L18.3818 34.3944C18.2406 34.4887 18.0674 34.545 17.8812 34.545L17.8806 34.5456ZM18.7893 28.1156V31.9456L21.6868 30.03L18.7893 28.1156Z" fill="#005AF0" />
+							</g>
+							<defs>
+								<clipPath id="clip0">
+									<rect width="40" height="40" fill="white" transform="translate(0.153076)" />
+								</clipPath>
+							</defs>
+						</svg>
+
+					</div>
+					<div className="welcome__section-description">
+						<h4>
+							{ __( 'Configure your site with AMP', 'amp' ) }
+						</h4>
+						<p dangerouslySetInnerHTML={ { __html: __( 'The AMP plugin for WordPress allows you to enable AMP content for your site. There are different ways this can be done, depending on the configuration of your site (e.g., which theme you are using) and your technical skills and level of interest in addressing AMP compatibility issues as your site evolves.', 'amp' ) } } />
```

Comment:
Similarly to above:

```suggestion
						<p>
							{ __( 'The AMP plugin for WordPress allows you to enable AMP content for your site. There are different ways this can be done, depending on the configuration of your site (e.g., which theme you are using) and your technical skills and level of interest in addressing AMP compatibility issues as your site evolves.', 'amp' ) }
						</p>
```